### PR TITLE
Added C# generation

### PR DESCRIPTION
**Pull request by michaeljmonte**
-Contributor License Agreement signed.
-Added C# generation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,8 @@ golang, and elixir:
 - golang: currently using a fork of go-xdr, but has complete support
 - elixir: support is experimental as the SDK is in early development. Generated
   code requires [:exdr](https://github.com/revelrylabs/exdr) in your deps
-
+- C#: seems complete, testing might prove otherwise.
```
Comment (leighmcculloch) [inline README.md:28]: If we think it is complete, let's say it's complete.
```suggestion
- C#: complete support
```

**Review by leighmcculloch — COMMENTED** [review]
This looks great, thank you @michaeljmonte.

I deleted one file that looked unnecessary to commit.

I'm going to run some tests with it, and if the tests look good, I'll return and approve and merge.

File: README.md (hunk 1 above)
Comment (michaeljmonte) [reply README.md:28]: Sounds good.

**Review by leighmcculloch — APPROVED** [review]
I ran this branch and regenerated the code for https://github.com/elucidsoft/dotnet-stellar-sdk/ and the resulting code came out identical, minus some whitespace differences, so I think this is good to merge.

@bartekn Could you also 👀 on accepting this new generator into this tool?

@fracek Does this also look good for you?

**Comment by fracek** [comment]
@leighmcculloch This looks good to me.

**Pull request merged**